### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by MatthewBlanski** [MatthewBlanski/ApexDocJS#7]
Audit the project for accessibility. The current HTML output uses lists, which assists its accessibility, but does not use aria for its responsive content. Appropriate heading usage and other accessibility principles should be applied throughout its output.
